scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/spel/Typer.scala

```diff
@@ -64,7 +64,7 @@ private[spel] class Typer(
     staticMethodInvocationsChecking: Boolean,
     classDefinitionSet: ClassDefinitionSet,
     evaluationContextPreparer: EvaluationContextPreparer,
-    methodExecutionForUnknownAllowed: Boolean,
+    anyMethodExecutionForUnknownAllowed: Boolean,
```

Comment:
the `methodExecutionForUnknownAllowed` is the name of feature flag in `ExpressionConfig`, so imo lets rename it also there and in it's java counterpart or lets leave the previous name unchanged everywhere.

Reply:
I made it like this because of: https://github.com/TouK/nussknacker/pull/7110#discussion_r1821356588
I would leave as it is

Reply:
Yep, take a look @raphaelsolarski how it loos without this:
if (methodExecutionForUnknownAllowed)
...
else // assume that is not allowed
...  // execute method for unknown
:)

Reply:
Agree, but why don't we change it in other places ("ExpressionConfig and friends")?